mk/config.mk

```diff
@@ -176,6 +176,26 @@ CFG_WITH_USER_TA ?= y
 # case you implement your own TA store
 CFG_REE_FS_TA ?= y
 
+# Support for loading user TAs from a special section in the TEE binary
+# Such TAs are available even before tee-supplicant is available (hence their
+# name), but note that many services exported to TAs may need tee-supplicant,
+# so early use is limited to a subset of the TEE Internal Core API (crypto...)
+# To use this feature, set EARLY_TA to the paths to one or more TA ELF file(s)
+# For example:
+#   EARLY_TA=path/to/8aaaf200-2450-11e4-abe2-0002a5d5c51b.stripped.elf
```

Comment:
From this description it is not clear whether one should define several EARLY_TA values (sic) each defining the full path of the TA elf file or if EARLY_TA is expected a unique directory path where one can store several TA ELF files to be embedded in the core binary.

As `EARLY_TA` is a configuration directive, I would suggest to prefix it `CFG_EARLY_TA_PATH`.